### PR TITLE
Add a sorting command to votour

### PR DESCRIPTION
**Pull request by ppedrot**
Typing `s` in votour will now sort the current objects by increasing size. A patch of mine I use from time to time so there is no point in gatekeeping this.

### CODE REVIEW

**Comment by ppedrot** [comment]
@coqbot run full ci

File: checker/votour.ml [hunk 1]
```diff
@@ -14,22 +14,29 @@ open Values
...
     match int_of_string l with
     | v ->
       if v < 0 || v >= max then
         let () =
           Printf.printf "Out-of-range input! (only %d children)\n%!" max in
         read_num max
-      else Some v
+      else CmdChild v
     | exception Failure _ ->
       Printf.printf "Unrecognized input! <n> enters the <n>-th child, u goes up 1 level, x exits\n%!";
```
Comment (SkySkimmer) [inline checker/votour.ml:41]: Missing `s sorts`?
Printing the commands should probably be shared with the other print at line 415(master)

Comment (ppedrot) [reply checker/votour.ml:41]: There should be a "help" command rather, but it's adding cruft atop of this PR. I'll do it.

Comment (ppedrot) [reply checker/votour.ml]: Done.

**Review by SkySkimmer — APPROVED** [review]

**Comment by SkySkimmer** [comment]
@coqbot merge now

**Pull request merged**